src/Algorithm.ts

```diff
@@ -87,13 +87,12 @@ export default class Algorithm extends Builder {
                 continue;
               }
               const completionyThing = part.contents.match(
-                /\b(ReturnIfAbrupt|throw|Return (Normal|Throw)?Completion|the result of evaluating)\b|(?<=[\s(])\?\s/i
+                /\b(ReturnIfAbrupt|(^|(?<=, ))[tT]hrow a|[rR]eturn (Normal|Throw|Return)?Completion\(|[rR]eturn( a| a new| the)? Completion Record|the result of evaluating)(\b|$)|(?<=[\s(])\?\s/
```

Comment:
```suggestion
                /\b(ReturnIfAbrupt\b|(^|(?<=, ))[tT]hrow (a\b|the\b|_|\*)|[rR]eturn (Normal|Throw|Return)?Completion\(|[rR]eturn( a| a new| the)?Completion Record\b|the result of evaluating\b)|(?<=[\s(])\?\s/
```

This will permit "throw the", "throw \_alias\_", "throw \*true\*", etc.

Reply:
This is running on the parsed ecmarkdown, not the plain text input, so matching `_` and `*` would require a much larger refactor which I don't really want to do. I could add `the` I guess? But in practice that hasn't come up.

Reply:
Though I guess actually `throw $` would match text followed by `*`, `~`, `_`, etc, which is probably good enough. Updated.